fido/fido.py

```diff
@@ -20,9 +19,46 @@
 
 
 DEFAULT_USER_AGENT = 'Fido/%s' % __about__.__version__
-
 DEFAULT_CONTENT_TYPE = 'application/json'
 
+# infinite timeouts are bad
```

Comment:
a more descriptive explanation here ? (like crochet defaulting to infinite timeout yadi yada ...)